crates/metaslang/bindings/src/lib.rs

```diff
@@ -54,8 +55,55 @@ pub struct Bindings<KT: KindTypes + 'static> {
     context: Option<GraphHandle>,
 }
 
+pub enum FileKind {
```

Comment:
`FileKind` doesn't indicate that it also holds the string identifier, but only a payload-less variant/kind. I wonder if something like `FileID` would better indicate this?

Reply:
Agreed. I'm not 100% convinced by `FileDescriptor` but I don't have a better alternative at this time, so I'll use it.